Add TiDB Cloud Terraform Doc (#10563)
pingcap/docs: tidb-cloud/terraform/destroy.md

---
title: Destroy resource
summary: Learn how to destroy resource
---

# Destroy Resource

> To destroy the resource, you can simply use `terraform destroy` and type `yes`. Don't worry about the order of deletion, terraform will generate a DAG based on the dependencies automatically.

Reviewer: ```suggestion
To destroy resources, you can simply run the `terraform destroy` command and type `yes`. Don't worry about the order of deletion, terraform will generate a DAG based on the dependencies automatically.
```

Reviewer: What is "DAG"?

Author: directed acyclic graph (DAG) is used to determine the dependency of resources. So, we can destroy them by order

> To destroy resources, you can simply run the `terraform destroy` command and type `yes`. Don't worry about the order of deletion, terraform will generate a DAG based on the dependencies automatically.

Author: 会调整一下

```
$ terraform destroy

Plan: 0 to add, 0 to change, 4 to destroy.

Do you really want to destroy all resources?
  Terraform will destroy all your managed infrastructure, as shown above.
  There is no undo. Only 'yes' will be accepted to confirm.

  Enter a value: yes

tidbcloud_cluster.restore_cluster1: Destroying... [id=1379661944630264072]
tidbcloud_cluster.restore_cluster1: Destruction complete after 2s
tidbcloud_restore.example_restore: Destroying... [id=780114]
tidbcloud_restore.example_restore: Destruction complete after 0s
tidbcloud_backup.example_backup: Destroying... [id=1350048]
tidbcloud_backup.example_backup: Destruction complete after 2s
tidbcloud_cluster.example_cluster: Destroying... [id=1379661944630234067]
tidbcloud_cluster.example_cluster: Destruction complete after 0s
╷
│ Warning: Unsupported
│ 
│ restore can't be delete
╵

Destroy complete! Resources: 4 destroyed.
```

> Note that a warning is appeared for restore can't be deleted.

Reviewer: ```suggestion
Note that you will get a warning indicating `restore can't be deleted`.
```

> If you execute `terraform show`, you will find nothing for all the states is cleared:

Reviewer: ```suggestion
Now, if you run the `terraform show` command, you will get nothing because all the states have been cleared:
```
